window.PorfolioConfig = {
  name: 'Alex Fernandes',
  jobTitle: 'Bioinformatician | Data Scientist | Medical Lab Tech',
  jobDescription: 'I am a dedicated bioinformatics and computational biology professional committed to advancing scientific research through innovative, data-driven solutions. With extensive experience developing and optimizing scalable pipelines for large-scale biological datasets, my expertise spans single-cell omics, next-generation sequencing (NGS) analysis, and high-performance computing (HPC). I have streamlined workflows and integrated machine learning techniques to transform complex data into actionable insights that drive research forward. Over the past 13 years in biomedical sciences — including 8+ years in bioinformatics, hands-on NGS, and high-dimensional data analysis — I have contributed to multiple research projects and authored over 12 peer-reviewed publications. My academic background, comprising a BSc in Biomedical Sciences and an MSc in Biochemistry and Molecular Biology, combined with research roles in Canada and Brazil, has sharpened my programming skills in Python, R, and Bash and enhanced my ability to visualize complex biological data effectively.',
  skills: [
    {
      name: 'Medical Biology',
      description: 'That is what drives my passion for Bioinformatics and Data Science. I have a BSc in Biomedical Sciences and a MSc in Biochemistry and Molecular Biology.'
    },
    {
      name: 'Python',
      description: ' My prefered general-purpose computer programming language to automate tasks, and conduct data analysis. My favorite libraries are Pandas, Scanpy, Plotly, Numpy, Scipy, Matplotlib, Seaborn and many others.'
    },
    {
      name: 'R',
      description: ' My prefered language for statistical computing and data visualization. It goes well with Python for data mining, bioinformatics and data analysis.'
    },
    {
      name: 'Bash',
      description: ' As scripting language and the best way to connect to HPC environments.'
    },
    {
      name: 'SQL',
      description: ' Mostly using PostgreSQL, this is my preffered programming language for storing and processing data in a relational databases.'
    },
    {
      name: 'Bioinformatics',
      description: ' RNA‑seq, single‑cell -omics, Next‑generation sequencing (NGS) analysis and High‑dimensional data visualization.'
    },
    {
      name: 'Data Science',
      description: ' Data integration, Data Mining, Statistics and, more recently, Machine Learning.'
    }
  ],
  works: [
    {
      name: 'The chromatin remodeler ADNP regulates neurodevelopmental disorder risk genes and neocortical neurogenesis',
      link: 'https://www.pnas.org/doi/abs/10.1073/pnas.2405981122',
      description: ' ADNP is a key subunit of the ChAHP chromatin remodeling complex and a high-confidence neurodevelopmental disorder (NDD) risk gene. However, the role of ADNP in neurodevelopment is unclear. Using a conditional mutant allele, we report that Adnp is required to expand upper-layer neurons during neocortical neurogenesis. By comparing Adnp and Chd4 conditional knockouts and by performing scRNA-seq and cut&run-seq, we further show that Adnp and Chd4 coregulate genes involved in cortical growth. Adnp additionally plays an important role in regulating NDD risk genes—particularly in upper-layer neocortical neurons. Our work provides insight into ChAHP functions during neocortical development.'
    },
    {
      name: 'Chd4 remodels chromatin to control retinal cell type specification and lineage termination',
      link: 'https://www.biorxiv.org/content/10.1101/2025.01.09.632287v2',
      description: ' During development, neural progenitor cells modify their output over time to produce different types of neurons and glia in chronological sequences. Previous studies have shown that epigenetic processes play a crucial role in regulating neural progenitor potential, but the underlying mechanisms are not well understood. Here, we hypothesized that nucleosome remodelling would regulate the competence transitions of retinal progenitors. We generated retina-specific conditional knockouts (cKOs) in the key nucleosome remodelling enzyme Chd4. Chd4 cKOs overproduced early-born retinal ganglion and amacrine cells. Postnatally, later-born rod photoreceptors were drastically underproduced. Concomitantly, progenitors failed to be exhausted at late phases of development and ultimately overproduced Müller glia. To determine how Chd4 regulates the genome, we used cut&run-seq to reveal Chd4 genome occupancy, and ATAC-seq experiments to visualize nucleosome remodelling. These data revealed that genome accessibility was significantly increased at ∼10,000 regulatory elements and ∼4,000 genes in the Chd4 cKO. Together, these results suggest that Chd4 restricts the genome to repress progenitor identity and promote rod photoreceptor production. Accordingly, multiplexed single-cell transcriptomics demonstrated that deletion of Chd4 led to markedly divergent gene expression profiles. However, despite overproduction of early fates and underproduction of later-born rods, the perinatal transition between early and late progenitor competence was not altered as determined by birthdating experiments and transcriptomic signatures. Taken together, our data suggest that Chd4-dependent chromatin remodelling regulates cell fate specification, and is also required to terminate retinal neurogenesis, but that it does not regulate the progenitor competence windows that restrict early- vs. late-cell-type production.'
    },
    {
      name: 'Competent to Stand Trial: The Case for Temporal Control of Retinal Development',
      link: 'https://www.sciencedirect.com/science/article/abs/pii/B9780443138201000530?via%3Dihub',
      description: ' The functions of the visual system depend on the precise wiring patterns of its constituent circuits. This in turn requires the ordered production of a constellation of different neuronal and glial cell types, each with unique morphologies, projection patterns, and physiology. During retinal development, this cellular diversity must be generated with chronological precision. To do this, retinal progenitor cells alter their behavior over developmental time—producing retinal cell types in a stereotyped sequence. However, the molecular mechanisms that temporally control progenitor competence are not fully understood. Here we review the evidence and concepts that underpin current thinking about retinal progenitor competence, and how this in turn relates to retinal lineage and cell fate determination.'
    }
  ],
  hobbies: [
    {
      name: 'MASTER OF SCiENCE (MSc) iN BiOCHEMiSTRY AND MOLECULAR BiOLOGY',
      description: ' Federal University of Rio Grande do Norte (UFRN), Natal, Brazil.'
    },
    {
      name: 'BACHELOR OF SCiENCE (BSc) iN BiOMEDiCiNE',
      description: ' Federal University of Rio Grande do Norte (UFRN), Natal, Brazil.'
    },
    {
      name: 'HiGH SCHOOL WiTH ASSOCiATE DEGREE iN INFORMATiCS AND PROGRAMMiNG',
      description: ' Federal Institute of Education, Science and Technology of Rio Grande do Norte (IFRN), Natal, Brazil.'
    }
  ],
  contacts: {
    email: 'aXXXf8a@icloXX.c0m',
    mobileNo: '+1 (X43) 5X2-6XX5',
    phoneNo: '+1 (613) 417-XXXX',
    links: [
      {
        name: 'Google Scholar',
        link: 'https://scholar.google.com/citations?user=cPPDy_wAAAAJ&hl=en'
      },
      {
        name: 'GitHub',
        link: 'https://github.com/alexfernandes8a'
      },
      {
        name: 'Twitter',
        link: 'https://twitter.com/alexfernandes8a'
      },
      {
        name: 'LinkedIn',
        link: 'https://www.linkedin.com/in/jos%C3%A9-alex-louren%C3%A7o-fernandes-71ba21115/'
      }
    ]
  }
};